### DISCUSSION TITLE
インシデントのコメントに画像を貼れるようにする

### DISCUSSION THREAD

**Discussion by gr1m0h** [topotal/waroom-roadmap#67] [Feature Request]
現状、インシデントのコメントに画像を貼ることができず、ダッシュボードのURLやSlackに貼った画像のURLを貼るしかない状況です

<img width="722" alt="スクリーンショット 2024-03-05 15 03 43" src="https://github.com/topotal/waroom-roadmap/assets/16541988/b5bd2ee5-779c-4b3c-8a9a-d371a3600933">

インシデント対応やPostmortemの際にWaroomで完結することが望ましいので、画像を貼れることを望みます

**Comment by sawa-zen** [comment]
@gr1m0h 
ありがとうございます！コメントだけでなく、ポストモーテムやステートドキュメント、ランブックにも画像を貼りたいという話は社内でも上がっているのでこちらもあらためて検討いたします。

おそらく実装する場合に段階的に実装することになると思うのですが、この「コメント、ステートドキュメント、ポストモーテム、ランブック」の4つの中でコメントが一番優先度が高いですか？もし理由などあれば教えていただけると助かります 🙏🏻

**Reply by gr1m0h** [reply]
@sawa-zen 
ありがとうございます！段階的に実装する旨も承知です。

優先度ですが、「ステートドキュメント >= ポストモーテム > コメント = ランブック」かと思いました。
ステートドキュメント、ポストモーテムは、情報をまとめる・後から振り返る時に情報の充実度が上がるので効果が高いと思います。
コメントとランブックは優劣判断できませんでした。
コメントは、Slackでインシデント対応チャンネルがあるのであれば、そちらに貼ってしまえば不要かなとは思いました。しかし、振り返る際にはarchiveされたSlackチャンネルを見に行くより、Waroomで完結した方が良い場面もあるかと思いました。（これはステートドキュメントの役割な気がするのでコメントの優先度は下げても良い気がしています。）
ランブックは、手順の上で画面のスクショがあると嬉しいケースがありそうですが、必須ではないので優先度下がるかなと思っています。

以上です。よろしくお願いします。

**Reply by sawa-zen** [reply]
やはり対応時と振り返り時に画像が貼れると情報共有しやすいのでステートドキュメントとポストモーテム周りが重要ですよね。コメント、ランブックに関しても承知しました 👍🏻 大変参考になります！ありがとうございます！

**Reply by sawa-zen** [reply]
以下の4箇所で画像の添付ができるようになりましたのでこちらクローズいたします。
- ステートドキュメント
- インシデントコメント
- ポストモーテム
- ポストモーテムコメント
